### ISSUE TITLE
PLE2502 documentation does not match the implementation

### ISSUE THREAD

**Issue by dscorbett** [astral-sh/ruff#14346]
The documentation for [`bidirectional-unicode` (PLE2502)](https://docs.astral.sh/ruff/rules/bidirectional-unicode/) says it checks for “bidirectional unicode characters”. That phrasing is imprecise (a string can be bidirectional but any given character isn’t) but the example code implies it means “right-to-left characters”. However, PLE2502 only checks for a selection of bidi control characters. Not all bidi control characters are right-to-left, and most right-to-left characters are ignored by the rule.

The documentation includes this example:
```python
s = "א" * 100  #  "א" is assigned
print(s)  # prints a 100-character string
```
However, there is no violation of PLE2502 in that example.
```console
$ ruff --version
ruff 0.7.3
$ ruff check --isolated --select PLE2502 ple2502.py
All checks passed!
```
